### PR TITLE
Save upload file

### PR DESCRIPTION
**Pull request by cjh5414**
- Add file folder
- Add uploaded file saving test
- Modify test_file location

### CODE REVIEW

File: kboard/board/tests/test_views.py [hunk 1]
```diff
@@ -310,3 +313,25 @@ def test_can_not_access_with_GET_on_delete_comment(self):
...
         self.assertEqual(response.status_code, 405)
+
+
+class FileUploadTest(BoardAppTest):
+    def test_save_uploaded_file(self):
+        uploaded_file_name = settings.BASE_DIR + '/test_file/test.txt'
+        saved_file_name = settings.BASE_DIR + '/file/test.txt'
+
+        if os.path.isfile(saved_file_name):
+            os.remove(saved_file_name)
```
Comment (guswnsxodlf) [inline kboard/board/tests/test_views.py]: Move down this `if` code to end of this functions

File: kboard/board/views.py [hunk 2]
```diff
@@ -3,19 +3,28 @@
...
 from board.forms import PostForm
 from core.utils import get_pages_nav_info
 
 
+def handle_uploaded_file(f):
+    with open(settings.BASE_DIR + '/file/' + f.name, 'wb+') as destination:
+        for chunk in f.chunks():
+            destination.write(chunk)
+
+
```
Comment (guswnsxodlf) [inline kboard/board/views.py:18]: Moving this function to different file will be better

File: kboard/board/tests/test_views.py (hunk 1 above)
Comment (cjh5414) [reply kboard/board/tests/test_views.py]: i wonder case that unreach end of code.

File: kboard/board/views.py (hunk 2 above)
Comment (cjh5414) [reply kboard/board/views.py:18]: i will move that function to forms.py after modelform refactoring.

File: kboard/board/tests/test_views.py [hunk 3]
```diff
@@ -310,3 +313,25 @@ def test_can_not_access_with_GET_on_delete_comment(self):
         )
 
         self.assertEqual(response.status_code, 405)
+
+
+class FileUploadTest(BoardAppTest):
+    def test_save_uploaded_file(self):
+        uploaded_file_name = settings.BASE_DIR + '/test_file/test.txt'
```
Comment (darjeeling) [inline kboard/board/tests/test_views.py]: naming fix please
IMHO `upload_test_file_name` is better

File: kboard/board/tests/test_views.py [hunk 4]
```diff
@@ -310,3 +313,25 @@ def test_can_not_access_with_GET_on_delete_comment(self):
...
+        response = self.client.post(reverse('board:new_post', args=[self.default_board.slug]), {
+            'post_title_text': 'NEW POST TITLE',
+            'fields': 'NEW POST CONTENT',
+            'file': uploaded_file,
+        })
+
+        uploaded_file.seek(0)
+        saved_file = open(saved_file_name)
+
+        self.assertEqual(uploaded_file.read(os.path.getsize(uploaded_file_name)),
```
Comment (darjeeling) [inline kboard/board/tests/test_views.py]: you don't need to pass filesize in python

File: kboard/functional_test/test_post_file_upload.py [hunk 5]
```diff
@@ -24,7 +24,7 @@ def test_file_upload(self):
 
         # 하단에 파일을 업로드 할 수 있는 버튼을 클릭하여 파일을 추가한다.
         fileuploadbox = self.browser.find_elements_by_tag_name('input')[2]
-        fileuploadbox.send_keys(settings.BASE_DIR + '/functional_test/test_file/test.txt')
+        fileuploadbox.send_keys(settings.BASE_DIR + '/test_file/test.txt')
```
Comment (darjeeling) [inline kboard/functional_test/test_post_file_upload.py]: please use `os.path.join`

**Comment by hyesun03** [comment]
LGTM

**Comment by guswnsxodlf** [comment]
LGTM

**Pull request merged**